dll/kernel32.cpp

```diff
@@ -477,7 +477,7 @@ namespace kernel32 {
 
 	LPWSTR WIN_FUNC GetCommandLineW() {
 		DEBUG_LOG("GetCommandLineW -> ");
-		return stringToWideString(GetCommandLineA());
+		return stringToWideString(GetCommandLineA()).data();
```

Comment:
use after free -- this needs a malloc'd copy, or caching the result of the conversion in a global variable forever